docfx_project/ziti/overview.md

```diff
@@ -5,9 +5,60 @@ uid: zitiOverview
 # Overview
 
 ## Why Ziti?
-Ziti represents the next generation of secure networking for your applications and has arrived with the Ziti platform,
-created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust
-entry point into your network or as a secure zero-trust proxy to other hosted services.
+Ziti represents the next generation of secure, open-source networking for your applications and has arrived with the Ziti platform, created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust entry point into your network or as a secure zero-trust proxy to other hosted services.
```

Comment:
The `and has arrived with the Ziti platform, created by Netfoundry, Inc !` seems wordy and forced. It doesn't have to change but I feel like that can be retooled. I'm not sure the quoted part needs to be in there at all? Maybe a separate section lower/at the bottom that goes over Netfoundry's involvement.

Reply:
Yeah, I was thinking the same thing. Don't want to strip NF out, but it's a bit jarring now and could be more subtle. Separate section at the bottom could be good, especially incorporate with something that says 'The Ziti project welcomes your contributions'